textbook/03/5/1/BuiltIn.ipynb

```diff
@@ -6,27 +6,287 @@
    "source": [
```

Comment:
<p>name for maximum is max -&gt; name for maximum is <code>max</code></p><p> </p><p> </p><p>The <em>help</em> function.  -&gt;. The <code>help</code> function.</p><p> </p><p> accepts a fuction  -&gt; accepts a function</p><p> </p><p>Calling the help function on max  -&gt; Calling the <code>help</code>  function on <code>max</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>